canonical-web-and-design/ubuntu.com
# Fix bug in advantage

## Done
Updated the design and UX of the SPA. Dynamically display the enterprise table when available.

## QA
- Go to the /advantage page
- See that page looks good
- Login
- See that the free token works and looks nice
-  Its difficult to get an enterprise contract so I will be asking for the JAAS team to QA that view

Fixes https://github.com/canonical-web-and-design/base-squad/issues/709
Fixes https://github.com/canonical-web-and-design/base-squad/issues/675
